ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -371,7 +372,12 @@ public void init() {
         "/history/:id",
         (req, res) -> {
           String id = req.params(":id");
-          List<Result> queryResponse = getMetacardHistory(id);
+          QueryParamsMap paramsMap = req.queryMap().get("sourceId");
```

Comment:
✏️ Use a path parameter instead of a query param. The `metacardId` and `sourceId` are a candidate key for the record and should both by query params because they are both needed to retrieve the resource.

https://stackoverflow.com/questions/30967822/when-do-i-use-path-params-vs-query-params-in-a-restful-api

Reply:
Right, but the sourceId acts as a filter, not a specific resource specifier